usr/share/rear/output/ISO/Linux-i386/850_check_for_errors.sh

```diff
@@ -1,2 +1,4 @@
 [[ $DEBUGSCRIPTS -eq 1 ]] && return   # to avoid throwing an error in DEBUG mode
 grep -iq "no space left" $RUNTIME_LOGFILE && Error "write error: No space left on device"
+# do not return 1 if grep fails - that's normal and expected
+return
```

Comment:
I think this must be an explicit `return 0`
because "help return" shows (excerpt)
```
return: return [n]
...
If N is omitted, the return status is that of the
last command executed within the function or script.
```

Reply:
Ooops, indeed, the message is still there: https://artifacts.dev.testing-farm.io/c14892c8-4ffa-47fc-900a-02c2a32f42d4/work-backup-and-restore3tq2lawj/tests/plans/backup-and-restore/execute/data/guest/default-0/make-backup-and-restore-iso-1/data/rear-mkbackup.log

Reply:
Fixed.